nucypher/types.py

```diff
@@ -30,7 +31,7 @@
 
 
 class ContractParams(Tuple):
-    pass
+    pass  # TODO
```

Comment:
Follow-up issue?

Reply:
@KPrasch

Reply:
For now, I suggest simply removing this type.

Reply:
✔️